benchmark_scenarios/worldcereal_crop_extent.json

```diff
@@ -36,12 +36,12 @@
       ]
     },
     "reference_data": {
-      "cropland-raw_2018-05-01_2019-04-30.tif": "https://s3.waw3-1.cloudferro.com/apex-benchmarks/worldcereal_crop_extent/cropland-raw_2018-05-01_2019-04-30.tif",
-      "job-results.json": "https://s3.waw3-1.cloudferro.com/apex-benchmarks/worldcereal_crop_extent/job-results.json",
-      "openEO.tif": "https://s3.waw3-1.cloudferro.com/apex-benchmarks/worldcereal_crop_extent/openEO.tif"
+      "worldcereal-cropland-extent.tif": "https://s3.waw3-1.cloudferro.com/swift/v1/APEx-benchmarks/worldcereal_crop_extent/worldcereal-cropland-extent.tif",
+      "job-results.json": "https://s3.waw3-1.cloudferro.com/swift/v1/APEx-benchmarks/worldcereal_crop_extent/job-results.json",
+      "worldcereal-cropland-extent-postprocessed.tif": "https://s3.waw3-1.cloudferro.com/swift/v1/APEx-benchmarks/worldcereal_crop_extent/worldcereal-cropland-extent-postprocessed.tif"
     },
     "reference_options": {
-      "atol": 1
+      "atol": 10
```

Comment:
isn't this a rather high tolerance for a discrete classification?

Reply:
Yes, so here we have a bit of a problem:
- The benchmarks compare all bands of the geotiff at once with one single tolerance level
- For the actual classification (0 or 1) this tolerance indeed doesn't make much sense, but also the classification probabilities are included in the geotiffs. These seemed to vary quite a lot, causing the benchmark to always fail with an absolute tolerance of 1